src/lib.rs

```diff
@@ -217,16 +234,64 @@ pub trait PolynomialCommitment<F: Field, P: Polynomial<F>>: Sized {
         Self::Commitment: 'a,
         P: 'a,
     {
-        let opening_challenges = |pow| opening_challenge.pow(&[pow]);
-        Self::batch_open_individual_opening_challenges(
-            ck,
-            labeled_polynomials,
-            commitments,
-            query_set,
-            &opening_challenges,
-            rands,
-            rng,
-        )
+        let rng = &mut crate::optional_rng::OptionalRng(rng);
```

Comment:
I think this is out of sync with the other PR, no? Namely we decided to have `*_individual_opening_challenges` be the methods that need to be implemented, and have everything else be implemented in terms of the latter?